Tests/DataTable/RegistryTest.php

```diff
@@ -18,7 +19,7 @@ public function testRetrieveTableById()
         $registry = new DataTableRegistry(['test' => $table]);
         $this->assertEquals($table, $registry->retrieveByTableId('test'));
 
-        $this->setExpectedException('\BadMethodCallException');
+        $this->expectException('\BadMethodCallException');
```

Comment:
```suggestion
        $this->expectException(BadMethodCallException::class);
```
Is it possible if we declare this class in the uses block and use ::class here?

Reply:
Done.